### DISCUSSION TITLE
precipitates

### DISCUSSION THREAD

**Discussion by siwux** [neperfepx/FEPX#95] [Q&A]
Hello, I saw the following formula in the FEPX web document considering precipitates, but did not give the source of the theory.Could you please inform me of the original theoretical literature?
![image](https://user-images.githubusercontent.com/90365358/186622943-e0a14e6e-ec29-47dd-88b2-1c476a00c77a.png)

**Comment by mkasemer** [comment]
This is a generalized form of a common equation used to estimate the increase in strength owing to the presence of precipitates. There are plenty of references, perhaps the best starting point would be the book "Mechanical Behavior of Materials" by Courtney.

Note that this equation works only until a certain average precipitate radius, afterward the contribution to strength begins to decrease - this is known as bowing, looping, or Orowan looping. We will be extending the model soon to include this feature.

**Reply by siwux** [reply]
Thanks a lot for your reply
